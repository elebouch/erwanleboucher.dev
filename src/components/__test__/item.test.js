import React from "react"
import renderer from "react-test-renderer"
import Item from "../item"

describe("Item", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Item title="test" description="test" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
