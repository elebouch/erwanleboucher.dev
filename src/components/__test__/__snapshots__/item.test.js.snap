// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item renders correctly 1`] = `"<div class=\\"sc-AxjAm cFQoNO\\"><p class=\\"sc-AxirZ eSQAuU\\">test</p><p class=\\"sc-AxiKw icjPNi\\">test</p></div>"`;
